tests/test_cnn.py

```diff
@@ -21,7 +21,7 @@ def test_cnn(tmp_path, model_class):
     # to check that the network handle it automatically
     env = FakeImageEnv(screen_height=40, screen_width=40, n_channels=1, discrete=model_class not in {SAC, TD3})
     if model_class in {A2C, PPO}:
-        kwargs = dict(n_steps=100)
+        kwargs = dict(n_steps=100, batch_size=100)
```

Comment:
This seems to break the test. I would fix this but I am not sure what the intention here was.

Reply:
I forgot A2C did not have a "batch_size" argument... this was to remove a warning about truncated mini-batch, I will change to `n_steps=64` instead